<?xml version="1.0" encoding="UTF-8"?>
<!-- SPDX-FileCopyrightText: 2025 zhang hongyuan <2063218120@qq.com> -->
<!-- SPDX-License-Identifier: MIT -->
<ui version="4.0">
 <class>PlayerWG</class>
 <widget class="QWidget" name="PlayerWG">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>758</width>
    <height>522</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Media Player</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="videoWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>320</width>
       <height>240</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: black;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="playProgressbar">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>6</height>
      </size>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
     <property name="textDirection">
      <enum>QProgressBar::BottomToTop</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="controlsWidget" native="true">
     <layout class="QVBoxLayout" name="controlsLayout">
      <item>
       <layout class="QHBoxLayout" name="buttonsLayout">
        <item>
         <widget class="QPushButton" name="playPauseBtn">
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Play</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="stopBtn">
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Stop</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="styleSheet">
           <string notr="true">image: url(:/128x128/volumn.png);
min-width: 28px;</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="volumeSpinBox">
          <property name="maximum">
           <number>200</number>
          </property>
          <property name="value">
           <number>50</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
